facilities/models.py

```diff
@@ -0,0 +1,83 @@
+import random
+from django.db import models
+from common.models import AbstractBase, SubCounty
+
+
+class Owner(AbstractBase):
+    name = models.CharField(max_length=100, unique=True)
+    description = models.TextField(null=True, blank=True)
+    code = models.CharField(max_length=100, unique=True)
+
+    def generate_code(self):
+        return "{}{}".format("OWNER", str(self.id))
```

Comment:
See previous comments about why this style of generating codes will not work

( TL;DR - at the point of `save`, we do not know the `id` yet; plus - `id`s will soon be UUIDs ).